h/services/search_index/_queue.py

```diff
@@ -61,10 +61,10 @@ def add_where(self, where, tag, priority, force=False, schedule_in=None):
             [Job.name, Job.scheduled_at, Job.priority, Job.tag, Job.kwargs],
             select(
                 [
-                    text("'sync_annotation'"),
-                    text(f"'{schedule_at}'"),
-                    text(str(priority)),
-                    text(repr(tag)),
+                    literal_column("'sync_annotation'"),
+                    literal_column(f"'{schedule_at}'"),
+                    literal_column(str(priority)),
+                    literal_column(repr(tag)),
```

Comment:
To test this you'd go to the admin pages and, for example, rename a user. It has to be a user who has multiple annotations. And test that the annotations do get updated with the new username